### PR TITLE
Name the Travis job for 'cargo fmt'

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ addons:
 matrix:
   include:
   - rust: stable
+    name: Format
```
Comment (datanel) [inline .travis.yml]: it does not format but check the formatting

Comment (woshilapin) [reply .travis.yml]: So `Format check`?

Comment (datanel) [reply .travis.yml]: formatting check? or check of formatting?
format sounds like paper format

Comment (woshilapin) [reply .travis.yml]: Changed to `Formatting check`.

**Pull request merged**